### PR TITLE
YJDH-106 | Fix kesäseteli review pipeline

### PR DESCRIPTION
**Pull request by klemmari1**

### CODE REVIEW

File: .github/workflows/ks-review.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# TODO: Fix this workflow
```
Comment (nenonja) [inline .github/workflows/ks-review.yml:1]: todo can be removed

**Review by nenonja — COMMENTED** [review]
Why frontend unit tests are not run at all? Perhaps because no changes to the frontend folder?
I can check it

File: .github/workflows/bf-review.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ on:
     paths:
     - 'backend/helsinki-benefit/**'
     - 'frontend/helsinki-benefit/**'
+    - 'frontend/shared/**'
+    - 'frontend/*'
```
Comment (quyenlq) [inline .github/workflows/bf-review.yml:8]: Hmn... Wouldn't this will trigger the Helsinki Benefit build when there is a change in `frontend/kesaseteli` ?

Comment (nenonja) [reply .github/workflows/bf-review.yml:8]: as far as I understand it should not include subfolders: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Comment (quyenlq) [reply .github/workflows/bf-review.yml:8]: Ah, yeah, now I get it :) Thanks for the clarification

**Review by nenonja — APPROVED** [review]
ks-empl keeps failing to infamous pages directory error but otherwise I think this is good now and ready to merge

**Pull request merged**